packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -131,6 +131,15 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
   private _scrollIdleTimeoutId: number | undefined;
 
+  // This tracks how the comboBox was opened
```

Comment:
Comment is a little misleading, it doesn't actually track anything about how the combo box was opened, it just says what should happen after close.

Reply:
true, I'll update the comment